### PR TITLE
Adding Sepolia as a default test network

### LINKED ISSUE #15750: Add Sepolia as a default test network
Sepolia is the new test network for Ethereum post merge and we should add it to our default test network list as 3 of the current test networks on that list are being deprecated (Related issue: https://github.com/MetaMask/metamask-extension/issues/15295). 

### Acceptance criterias:
- The network uses Infura's endpoint as we always want to use rpc endpoints that we know comply to our privacy policy.
- The Infura key should not be revealed to users when users review the details for the Sepolia network in Settings > Networks.

### PR DESCRIPTION
**Pull request by ryanml**
Related: https://github.com/MetaMask/metamask-extension/issues/15750

Screen Shot 2022-09-12 at 12 14 44 PM

Screen Shot 2022-09-12 at 12 14 37 PM

Screen Shot 2022-09-12 at 12 14 29 PM

### CODE REVIEW

File: ui/components/app/loading-network-screen/loading-network-screen.component.js [hunk 1]
```diff
@@ -51,6 +51,8 @@ export default class LoadingNetworkScreen extends PureComponent {
       name = this.context.t('connectingToRinkeby');
     } else if (providerName === 'goerli') {
       name = this.context.t('connectingToGoerli');
+    } else if (providerName === 'sepolia') {
```
Comment (darkwing) [inline ui/components/app/loading-network-screen/loading-network-screen.component.js:54]: We should eventually change this to a `switch`

File: ui/pages/settings/networks-tab/networks-tab.constants.js [hunk 2]
```diff
@@ -51,6 +53,15 @@ const defaultNetworksData = [
     ticker: TEST_NETWORK_TICKER_MAP[GOERLI],
     blockExplorerUrl: 'https://goerli.etherscan.io',
   },
+  {
+    labelKey: SEPOLIA,
+    iconColor: '#CFB5F0',
```
Comment (darkwing) [inline ui/pages/settings/networks-tab/networks-tab.constants.js:58]: @georgewrmarshall Do you know if we can use `var` colors for these variables?

Comment (georgewrmarshall) [reply ui/pages/settings/networks-tab/networks-tab.constants.js:58]: Nice catch @darkwing! It's in the works https://github.com/MetaMask/design-tokens/issues/238. I can update this once we have updated the design tokens repo :)

**Review by danjm — DISMISSED** [review]
LGTM

**Review by Gtonizuka — DISMISSED** [review]

**Comment by brad-decker** [comment]
This got out of date due to networks constants typescript conversion. I'm going to update it.

**Comment by brad-decker** [comment]
@danjm @Gtonizuka @ryanml updated to use the new network constants

**Review by jpuri — APPROVED** [review]
LGTM

**Review by brad-decker — APPROVED** [review]

**Comment by ryanml** [comment]
Thanks @brad-decker!

**Pull request merged**